torch/distributed/rendezvous.py

```diff
@@ -90,7 +90,7 @@ def _rendezvous_helper(url: str, rank: int, world_size_opt: Optional[int], **kwa
 
 
 def rendezvous(url: str, rank: int = -1, world_size: int = -1, **kwargs):
-    if not isinstance(url, str):
+    if not isinstance(url, (str, bytes)):
         raise RuntimeError("`url` must be a string. {}: {}".format(type(url), url))
```

Comment:
This should probably be a TypeError. Also the functions signature of rendezvous is now wrong (url should be a Union[str, bytes] right?)

Reply:
Yes, can you open an issue so that we can ask the distributed team what they want to do about that?